ice/trace.py

```diff
@@ -116,6 +117,12 @@ def emit_block(x) -> tuple[int, int]:
         return 0, 0
 
 
+def add_fields(**fields: str):
```

Comment:
Not critical, but I'd like to come up with a better name for this function before releasing it.